templates/server/src/main/resources/archetype-resources/core/src/test/java/__packageInPathFormat__/general/common/base/ApplicationTest.java

```diff
@@ -0,0 +1,22 @@
+package ${package}.general.common.base;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+
+import com.devonfw.module.basic.common.api.reflect.Devon4jPackage;
+import com.devonfw.module.test.common.base.ModuleTest;
+import com.example.application.sampleapp.SpringBootApp;
+import com.example.application.sampleapp.general.common.base.Devon4jPackageCheckTest;
+
+/**
+ * This test verifies that {@link SpringBootApp} is able to startup.
+ */
+public class ApplicationTest extends ApplicationComponentTest {
```

Comment:
`ApplicationComponentTest` is from sub-package `test`. I have no clue, how this could ever work without import.